Purchases/Public/RCPurchases.m

```diff
@@ -1166,5 +1178,15 @@ - (void)postReceiptWithTransaction:(SKPaymentTransaction *)transaction
                        }];
 }
 
+- (nullable NSString *)productIdentifierFrom:(SKPaymentTransaction *)transaction
+{
```

Comment:
nitpick: the `{` should go at the end, 
`- (nullable NSString *)productIdentifierFrom:(SKPaymentTransaction *)transaction {`